website/client/js/controllers/menuCtrl.js

```diff
@@ -29,6 +29,32 @@ angular.module('habitrpg')
       $scope.clearMessages = Chat.markChatSeen;
       $scope.clearCards = Chat.clearCards;
 
+      $scope.getNotificationsCount = function() {
+        var count = 0;
+
+        if($scope.user.invitations.party && $scope.user.invitations.party.id){
+          count++;
+        }
+
+        if($scope.user.purchased.plans && $scope.user.purchased.plans.mysteryItems.length){
```

Comment:
You have a typo here. It's `plan` not `plans`. I've fixed it up and squashed the commits.